openhtf/exe/phase_data.py

```diff
@@ -72,9 +72,7 @@ def _asdict(self):
     """Return a dict of this PhaseData's public data."""
     return {'measurements': self.measurements,
             'attachments': self.attachments.keys(),
-            'plugs': {
-                k.__module__ + '.' + k.__name__: str(v)
-                for k, v in self.plug_map.iteritems()}}
+            'plug_manager': self.plug_manager}
```

Comment:
hm, i forget why this even needed an `_asdict` in the first place.  if possible, we should get rid of this.  if it's needed, then I'm guessing it's somewhere that expects all base types, so i'd rather keep the `plugs` dict and access some helper method on `self.plug_manager` to pull the strings out that we used to generate directly on `self.plug_map`